src/core/maprenderer/qgsmaprenderersequentialjob.cpp

```diff
@@ -29,8 +29,8 @@ QgsMapRendererSequentialJob::QgsMapRendererSequentialJob( const QgsMapSettings &
 
   mImage = QImage( mSettings.deviceOutputSize(), mSettings.outputImageFormat() );
   mImage.setDevicePixelRatio( mSettings.devicePixelRatio() );
-  mImage.setDotsPerMeterX( mSettings.devicePixelRatio() * 1000 * settings.outputDpi() / 25.4 );
-  mImage.setDotsPerMeterY( mSettings.devicePixelRatio() * 1000 * settings.outputDpi() / 25.4 );
+  mImage.setDotsPerMeterX( 1000 * settings.outputDpi() / 25.4 );
```

Comment:
how has this change been tested? My understanding is that @3nids added this to fix rendering on mac

Reply:
For this one, I have a separated PR ready with tests that I need to push!